css/sponsors.css

```diff
@@ -160,3 +172,6 @@ h1 {
     font-weight: 600;
     color: #666;
 }
+h1.col-md-12{
+    padding-bottom: 50px;
+}
```

Comment:
@loicmurumba oh no it's a standard to leave a new line at the end of a css file. See the red mark at the bottom of the github change page?